modules/standard/Math.chpl

```diff
@@ -1290,6 +1290,22 @@ module Math {
     extern proc yn(n: c_int, x: real(64)): real(64);
     return yn(n.safeCast(c_int), x);
   }
+  
+  /* Returns 1 if the sign of `x` is negative, else returns 0. It detects 
```

Comment:
The C version says `The signbit() macro returns nonzero if the sign of x is negative; otherwise it returns zero.` The difference being that the C version can return any non-0 value if x is negative. You'll either need to update our documentation (and the test, which currently fails for me) or force this function to convert non-0 values to `1`

Reply:
FYI the [C++ version](https://en.cppreference.com/w/cpp/numeric/math/signbit) returns a bool -- `true if x is negative, false otherwise`

I think an open question is whether we should follow the C or the C++ version. I would lean towards the C++ interface personally, but this isn't an area I'm familiar with.